server/src/test/java/io/spine/server/storage/memory/OperatorComparatorsShould.java

```diff
@@ -20,26 +20,18 @@
 
 package io.spine.server.storage.memory;
 
-import io.spine.base.EventId;
-import io.spine.server.entity.EntityRecord;
-import io.spine.server.event.EventStreamQuery;
-import io.spine.server.storage.EventRecordStorage;
-import io.spine.server.storage.RecordStorage;
+import io.spine.server.storage.memory.ComparisonOperations;
+import org.junit.Test;
 
-import java.util.Map;
+import static io.spine.test.Tests.assertHasPrivateParameterlessCtor;
 
 /**
- * {@inheritDoc}
+ * @author Dmytro Dashenkov
  */
-public class InMemoryEventRecordStorage extends EventRecordStorage {
+public class OperatorComparatorsShould {
```

Comment:
`OperatorEvaluatorsShould`.

Also let's make sure we apply nullability tests to the static classes like this one.